ArtemisKit/Sources/Messages/Views/ConversationView/ConversationInfoSheetView.swift

```diff
@@ -59,6 +60,24 @@ struct ConversationInfoSheetView: View {
             } content: {
                 CreateOrAddToChatView(courseId: viewModel.course.id, configuration: .addToChat(conversation))
             }
+            .alert(isPresented: $showDeleteConfirmation) {
+                Alert(
+                    title: Text(R.string.localizable.confirmDeleteTitle()),
+                    message: Text(String(format: R.string.localizable.confirmDeleteMessage(conversation.baseConversation.conversationName))),
+                    primaryButton: .destructive(Text(R.string.localizable.delete())) {
+                        viewModel.isLoading = true
+                        Task {
+                            let success = await viewModel.deleteChannel()
+                            if success {
+                                navigationController.goToCourseConversations(courseId: viewModel.course.id)
```

Comment:
```suggestion
                                navigationController.selectedPath = nil
```
This is a bit more efficient, goToCourseConversations will cause the whole course object to be reloaded